source/_posts/2021-02-03-release-20212.markdown

```diff
@@ -45,122 +54,247 @@ Issues introduced in the beta are processed with priority.
+integration: It is decoupled from Home Assistant. Instead of MQTT, the Z-Wave
+JS integration uses a WebSocket connection to a Z-Wave JS server.
+
+This means, in order to use this new integration, you'll need to run the
+Z-Wave JS server that sits in between your Z-Wave USB stick and Home Assistant.
+There are multiple options available on running the Z-Wave JS server,
+via Docker, manually, but there is also an Home Assistant add-on available.
+
+If you run an installation with a Supervisor, the add-on will even be
+set up automatically for you.
+
+[zwavejs]: https://zwave-js.github.io/node-zwave-js/#/
+
+### Current state of Z-Wave JS
+
+The new Z-Wave JS integration is amazingly complete. It supports the following
+platforms from the start: Binary Sensors, Climate, Covers, Fans, Lights, Locks,
+Sensors and Switches. Scenes are also supported!
+
+Besides that, it is just _blazing_ fast! ⚡️
+
+While fast-growing, it is a new integration and there is still a lot
+in progress. Of course, no doubt, there will be bugs. Bugs that will
+be squashed and handled. Please, be sure to
+[report issues in our issue tracker][issues].
+
+What is left? See our [Z-Wave JS integration roadmap][roadmap]
+for those details.
+
+[roadmap]: https://github.com/home-assistant-libs/zwave-js-server-python/issues/56
+[issues]: https://github.com/home-assistant/core/issues/new/choose
+
+### Thanks to all
+
+Thanks to all of the following people, the Z-Wave JS integration was able
+to see the light in just a month!
+
+- [@AlCalzone]
+- [@balloob]
+- [@bramkragten]
+- [@cgarwood]
+- [@firstof9]
+- [@itewk]
+- [@jcam]
+- [@kpine]
+- [@marcelveldt]
+- [@MartinHjelmare]
+- [@Michsior14]
+- [@Petro31]
 - [@pvizeli]
-[@cgarwood], [@MartinHjelmare], [@marcelveldt], [@raman325], [@firstof9], [@balloob]
--->
+- [@raman325]
+- [@robertsLando]
 
-Long story short: You are not going to believe this, but a complete new,
-full blown, integration was created from nothing in just a month!
+Yes, that is an awful lot of people! Amazing how this all came together
+and incredible work!
 
-Z-Wave JS is here!
+[@AlCalzone]: https://github.com/AlCalzone
+[@itewk]: https://github.com/itewk
+[@jcam]: https://github.com/jcam
+[@kpine]: https://github.com/kpin
+[@Michsior14]: https://github.com/Mchsior14
+[@robertsLando]: http://github.com/robertsLando
+
+### What about the old Z-Wave integration?
+
+The old [`zwave`](/integrations/zwave) integration has been based on
+OpenZWave 1.4, which is really old and does not get any upstream updates
+anymore. This also means that it is hard for us to keep it in a working state
+for the upcoming future.
+
+Therefore, as of today, the old Z-Wave is marked as being **deprecated**.
+
+If one is starting with Home Assistant and Z-Wave, the `zwave` integration is
+no longer recommended. For existing users, we recommend migrating to the
+[Z-Wave JS integration](/integrations/zwave_js). However, don't panic,
+`zwave` is deprecated but not removed yet.
```

Comment:
Maybe clarify that we don't plan on removing the `zwave` integration unless it can no longer be made to be build for the latest Python release. Now "yet" leaves a lot of room for interpretation.